src/config/components/OverlayAdmin/OverlayAdmin.tsx

```diff
@@ -187,14 +215,76 @@ export default class OverlayAdmin extends React.Component<Props> {
 								</li>
 							) : null}
 							<li>
+								<SectionHeader>Bob's Buddy</SectionHeader>
+								<label>
+									<input
+										type="checkbox"
+										checked={bobsBuddyEnabled}
+										disabled={this.props.disabled}
+										onChange={this.setFeature(Feature.BOBSBUDDY)}
+									/>{" "}
+									Enable Bob's Buddy
+								</label>
+							</li>
+							{bobsBuddyEnabled ? (
+								<li>
+									<p style={{ fontStyle: "italic" }}>
+										Viewers can see your Bob's Buddy results. They can move and
+										hide the panel for themselves. Note: This feature requires
+										Bob's Buddy to be active on your Hearthstone Deck Tracker.
```

Comment:
Viewers can no longer move the panel. We should also include where in HDT this setting is.